Kernel/Storage/StorageManagement.cpp

```diff
@@ -24,7 +24,7 @@
 namespace Kernel {
 
 static Singleton<StorageManagement> s_the;
-static Atomic<size_t> s_device_minor_number;
+static Atomic<u32> s_device_minor_number;
```

Comment:
I'm not sure that this change helps, but it does clarify that we hold a `u32` value. A `size_t` type is quite vague to be specified here.